### PR TITLE
Mehrdad Metadata ET

### PR DESCRIPTION
**Pull request by mehrdadhalali**
I have written a script to extract and transform the plant metadata, the upload has to be done manually, perhaps we could automate this later.

### CODE REVIEW

File: collect_metadata/extract.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        "timezone": timezone
+    }
+
+
+def extract_plant_data(reading_data: dict) -> dict:
+    """Returns the relevant metadata of the plant."""
+
+    image_data = reading_data.get("images", {})
+    plant_data = {
+        "plant_id": reading_data["plant_id"],
```
Comment (Leon-JS02) [inline collect_metadata/extract.py]: Should there be consistency between `dict[]` and `dict.get()`

File: collect_metadata/extract.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    plants = get_plant_data(50)
+
+    plant_data = []
+    for i in range(0, len(plants)):
+        plant_data.append(extract_relevant_data(plants[i]))
+
+    return plant_data
+
+
+if __name__ == "__main__":
```
Comment (Leon-JS02) [inline collect_metadata/extract.py]: Main is empty

File: collect_metadata/test_metadata.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        }
+
+        data = extract_botanist_data(raw_data)
+
+        assert data["first_name"] == "Eliza"
+        assert data["last_name"] == "Andrews"
+        assert data["email"] == "eliza.andrews@lnhm.co.uk"
+        assert data["phone"] == "(846)669-6651x75948"
+
+    def test_get_botanist_data_middle_name(self):
```
Comment (Leon-JS02) [inline collect_metadata/test_metadata.py:23]: I like this edge case!

File: short_term/extract.py [hunk 4]
```diff
@@ -0,0 +1,7 @@
+"""This script is for getting the plant data from the API."""
+import requests as req
+
+
+if __name__ == "__main__":
```
Comment (Leon-JS02) [inline short_term/extract.py]: Empty main

**Review by Leon-JS02 — CHANGES_REQUESTED** [review]
Mostly all good, just a few clean ups needed.

**Review by olliejth — APPROVED** [review]

**Review by Leon-JS02 — APPROVED** [review]
Thank you for making changes! A solid extract script.

**Pull request merged**